src/files/utils/spawn.ts

```diff
@@ -14,3 +14,17 @@ export const spawnPromiseAndCapture = async (command: string, args: string[], op
   }
   return [Buffer.concat(stdout), Buffer.concat(stderr), error];
 };
+
+export const escapeShellArguments = (args: string[]): string[] => {
+  //there is a distinct lack of cross-platform implementations of shell escaping.
+  //this is save and throws an error if it doesn't know what to do
+  return args.map(value => {
+    if(value.indexOf(' ')) {
```

Comment:
This should be `value.indexOf(' ') > -1`

Reply:
uups. Good catch